lib/grinder_sdk.dart

```diff
@@ -305,7 +306,7 @@ class Analyzer {
     List args = [];
     if (packageRoot != null) args.add('--package-root=${packageRoot.path}');
     if (fatalWarnings) args.add('--fatal-warnings');
-    args.addAll(_coerceToPathList(fileOrPaths));
+    args.addAll(coerceToPathList(fileOrPaths));
```

Comment:
It's a bit weird that here are `fileOrPaths`, `files` (line 320), `fileOrPath` (line 335) for the same kind of argument. Also the doc-comments are written differently.